easybuild/easyconfigs/m/MariaDB/MariaDB-10.3.10-foss-2018b.eb

```diff
@@ -38,6 +35,9 @@ checksums = [
     '8295837e623f6c782e1d64b00e0877ea98cce4bf8846755bb86c8a7732797c19',  # MariaDB-10.1.13-link-rt-for-jemalloc.patch
 ]
 
+# MariaDB pulls in submodules which don't have releases.
+# We download the submodule revisions from their repos.
+# determine commit of additional sources via https://github.com/MariaDB/server/tree/10.3
```

Comment:
@vanzod Please move this back up?

Reply:
This was inadvertently modified by `inject-checksum`. Reverting it back as it was.